.travis.yml

```diff
@@ -0,0 +1,14 @@
+language: cpp
+
+os:
+- linux
+- osx
+
+sudo: true
+
+before_script:
+- sh install-deps.sh
+
+script:
+- cmake .
```

Comment:
Why don’t you use the same method as appveyor, that is to build from a `build` directory?